lib/colorls/yaml.rb

```diff
@@ -0,0 +1,21 @@
+module ColorLS
+  class Yaml
+    def self.load_from_yaml(filename, aliase=false)
```

Comment:
Let's do a proper class instance variable.

```ruby
module ColorLS
  class YAML
    def initialize(filename)
      @filepath             = File.join(File.dirname(__FILE__),"../yaml/#{filename}")
      @user_config_filepath = File.join(Dir.home, ".config/colorls/#{filename}")
    end

    def load(aliase: false)
      yaml = read_file(@filepath)

      if File.exist?(@user_config_filepath)
        user_config_yaml = read_file(@user_config_filepath)
        yaml             = yaml.merge(user_config_yaml)
      end

      return yaml unless aliase
      yaml.to_a.map! { |k, v| [k, v.to_sym] }.to_h
    end

    private

    def read_file(filepath)
      YAML.safe_load(File.read(filepath)).symbolize_keys
    end
  end
end
```

So that, it can be used like -

```ruby
@opts[:colors] = ColorLS::YAML.new(color_scheme_file).load(aliase: true)
```

Reply:
Ping @alexpjohnson